js/main.js

```diff
@@ -1,382 +0,0 @@
-// Weather App
-// by Roger Mullins
-
-// Object Definitions
-
-let stateObj = {
-  city : "",
-  zipCode: "",
-  tempK: 0,
-  tempF: 0,
-  tempC: 0,
-  condition: "",
-  imageURL: ""
-};
-
-let lookupHistoryObj = {
-  city: "",
-  zipCode: ""
-};
-
-let localStorageObj = {
-  city: "",
-  zipCode: ""
-}
-
-// Global Variable
-
-let zipIn = "";   // Used to validate ZIP
-let tempF = 0;    // Used to update temperature background color
-
-function buildUI() {
-  // Creates HTML elements
-
-  function createElement(parentID, typeOfElement, bootstrapClasses, idString, innerText = "") {
-    let target = document.getElementById(parentID);
-    let newElement = document.createElement(typeOfElement);
-    newElement.classList.add(...bootstrapClasses);
-    newElement.setAttribute("id", idString);
-    if (innerText != "") {
```

Comment:
Technically an empty `""` is falsy so you could just simply write this as `if (innerText)`